internal/BUILD.bazel

```diff
@@ -70,10 +73,14 @@ ts_library(
 nodejs_binary(
     name = "tsc_wrapped_bin",
     data = [
+        "@npm//:protobufjs",
+        "@npm//:source-map-support",
+        "@npm//:tsickle",
```

Comment:
tsickle is required. hmmm.

Reply:
maybe add a TODO, I'm not that worried about it

Reply:
kk